source/installation-guide.rst

```diff
@@ -104,20 +119,20 @@ Installation
 To setup the environment, merely clone the deployment repository and run the
 commands below.
 
-The docker-compose enabled deployment scripts and configuration repository can
+The Docker Compose enabled deployment scripts and configuration repository can
 be found at:
 
 https://github.com/dojot/docker-compose
 
-or as git clone command:::
+or as git clone command: ::
 
   git clone https://github.com/dojot/docker-compose.git
   # Let's move into the repo - all commands in this page should be executed
   # inside it.
   cd docker-compose
 
 Once the repository is properly cloned, select the version to be used by
-checking out the appropriate tag (do notice that the tagname has to be
+checking out the appropriate tag (do notice that the tag_name has to be
```

Comment:
maybe it’s better to format the word as code (wrapping the word between two inverted single quotes: \``word\``):
> checking out the appropriate tag (do notice that the `tag_name` has to be